### DISCUSSION TITLE
How I can convert mbp3 or pksim5 files in SBML files?

### DISCUSSION THREAD

**Discussion by DBgentech2023sirius** [Open-Systems-Pharmacology/Forum#1739] [Questions & Problems] [answered]
Dear open-systems-pharmacology community,
I have recently started working with PK-Sim and Mobi software. I downloaded model in pksim5 format. I know that I can import and export json files from PK-Sim, and also I can import SBML files in Mobi. However for my research I need this model in SBML format. Please, help: how I can convert Mobi or PK-Sim project files in SBML file?

**Comment by Yuri05** [comment, marked as answer]
Currently, there is no automated process for converting PK-Sim/MoBi projects or simulations to SBML.
